.travis.yml

```diff
@@ -24,26 +24,44 @@ env:
 matrix:
   fast_finish: true
   include:
-# 7.3
     - name: 'Unit tests'
       php: 7.3
       env: TEST_CONFIG="phpunit.xml"
+    - name: 'Unit tests (PHP7.4)'
+      php: 7.4
+      env: TEST_CONFIG="phpunit.xml"
     - name: "Kernel Behat Core tests"
       php: 7.3
       env:
         - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
         - BEHAT_OPTS="--profile=core --tags=~@broken"
         - APP_ENV=behat
         - APP_DEBUG=1
+    - name: "Kernel Behat Core tests (PHP 7.4)"
```

Comment:
I'd remove this job as well

Reply:
Fixed in https://github.com/ezsystems/ezplatform-kernel/pull/37/commits/af8b9ca0a48aaf5805e90034d607f7e0d16120dc